src/ioctls/vcpu.rs

```diff
@@ -166,6 +166,12 @@ pub enum VcpuExit<'a> {
     X86Rdmsr(ReadMsrExit<'a>),
     /// Corresponds to KVM_EXIT_X86_WRMSR.
     X86Wrmsr(WriteMsrExit<'a>),
+    /// Corresponds to KVM_EXIT_MEMORY_FAULT.
+    MemoryFault(
+        u64, /* flags */
+        u64, /* gpa */
+        u64, /* size */
+    ),
```

Comment:
Let's use named fields here 

```suggestion
    MemoryFault {
        flags: u64,
        gpa: u64,
        size: u64,
    },
```

Reply:
done